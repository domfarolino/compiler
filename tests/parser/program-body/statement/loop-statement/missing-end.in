program dominic is

  integer dominic;
  integer n;

begin

  n := 10;
  for (dominic := 0; dominic < n)
    dominic := dominic + 1;
  ;

end program.
