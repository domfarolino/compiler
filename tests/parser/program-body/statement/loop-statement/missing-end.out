[1;31mLine 11: Expected 'end for' after for loop statement, but got: ';'
Line 11: Error parsing program body
[0m